src/main/resources/db/migration_sqlite/V1__initial.sql

```diff
@@ -1,153 +1,708 @@
-CREATE INDEX escrow_sender_id_height_idx ON escrow (sender_id, height DESC);
-CREATE INDEX escrow_recipient_id_height_idx ON escrow (recipient_id, height DESC);
-CREATE INDEX escrow_deadline_height_idx ON escrow (deadline, height DESC);
-CREATE TABLE escrow_decision (db_id INTEGER PRIMARY KEY AUTOINCREMENT, escrow_id BIGINT NOT NULL, account_id BIGINT NOT NULL,
-    decision INT NOT NULL, height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE);
-CREATE UNIQUE INDEX escrow_decision_escrow_id_account_id_height_idx ON escrow_decision (escrow_id, account_id, height DESC);
-CREATE INDEX escrow_decision_escrow_id_height_idx ON escrow_decision (escrow_id, height DESC);
-CREATE INDEX escrow_decision_account_id_height_idx ON escrow_decision (account_id, height DESC);
-CREATE TABLE subscription (db_id INTEGER PRIMARY KEY AUTOINCREMENT, id BIGINT NOT NULL, sender_id BIGINT NOT NULL, recipient_id BIGINT NOT NULL,
-    amount BIGINT NOT NULL, frequency INT NOT NULL, time_next INT NOT NULL, height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE);
-CREATE UNIQUE INDEX subscription_idx ON subscription(id, sender_id, recipient_id, amount, frequency, time_next, height, latest);
-CREATE UNIQUE INDEX subscription_id_height_idx ON subscription (id, height DESC);
-CREATE INDEX subscription_sender_id_height_idx ON subscription (sender_id, height DESC);
-CREATE INDEX subscription_recipient_id_height_idx ON subscription (recipient_id, height DESC);
-CREATE UNIQUE INDEX block_timestamp_idx ON block (timestamp DESC);
-CREATE TABLE at (db_id INTEGER PRIMARY KEY AUTOINCREMENT, id BIGINT NOT NULL, creator_id BIGINT NOT NULL, name VARCHAR, description VARCHAR,
-    version SMALLINT NOT NULL, csize INT NOT NULL, dsize INT NOT NULL, c_user_stack_bytes INT NOT NULL, c_call_stack_bytes INT NOT NULL,
-    creation_height INT NOT NULL, ap_code BLOB NOT NULL,
-    height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE);
-CREATE UNIQUE INDEX at_id_height_idx ON at (id, height DESC);
-CREATE INDEX at_creator_id_height_idx ON at (creator_id, height DESC);
-CREATE TABLE at_state (db_id INTEGER PRIMARY KEY AUTOINCREMENT, at_id BIGINT NOT NULL, state BLOB NOT NULL, prev_height INT NOT NULL,
-    next_height INT NOT NULL, sleep_between INT NOT NULL,
-    prev_balance BIGINT NOT NULL, freeze_when_same_balance BOOLEAN NOT NULL, min_activate_amount BIGINT NOT NULL, height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE);
-CREATE UNIQUE INDEX at_state_at_id_height_idx ON at_state (at_id, height DESC);
-CREATE INDEX at_state_id_next_height_height_idx ON at_state (at_id, next_height, height DESC);
-CREATE INDEX account_id_balance_height_idx ON account (id, balance, height DESC);
-CREATE INDEX transaction_recipient_id_amount_height_idx ON "transaction" (recipient_id, amount, height);
-CREATE INDEX account_id_latest_idx ON account(id, latest);
+-- block
+-- no generator_id foreign key because account may be not yet created
```

Comment:
Should be able to guarantee this.

Reply:
bb98740d19c7fff9d06a0347048fd1adb4dad256 should allow a foreign key here

Reply:
Thanks will check and add fg